### PR TITLE
Feat(VETS-CPC-728): get number of vet ratings for each vet

### PR DESCRIPTION
**Pull request by KarinaSofia**
JIRA: https://champlainsaintlambert.atlassian.net/browse/CPC-728?atlOrigin=eyJpIjoiNmQ3OTU5Y2VkNmU3NDk3NGE2MDBlZWQzZjExYmQyYzQiLCJwIjoiaiJ9
## Context:
We want to display the total number of ratings each vet has. Doing this allows the user to quickly see how many ratings a certain vet has. 
## Changes
- Implementation of a get number of ratings per vet in Api-Gateway
- Implementation of get number of ratings per vet in the Vet-Service
- 90% + testing in Api-gateway
- 90% + testing in Vet-Service
- Front-end Implementation: now displays the number of ratings per vet next to the vet's information
## Before and After UI (Required for UI-impacting PRs)
image
image

### CODE REVIEW

File: api-gateway/src/main/resources/static/scripts/vet-list/vet-list.controller.js [hunk 1]
```diff
@@ -30,10 +30,40 @@ angular.module('vetList')
             child.classList.add("modalOff");
         }
 
+        /*$http.get('api/gateway/vets').then(function (resp) {
```
Comment (DenisaNicoletaH) [inline api-gateway/src/main/resources/static/scripts/vet-list/vet-list.controller.js:33]: I would recommend removing unnecessary comments or code

**Review by DenisaNicoletaH — APPROVED** [review]
Other than the previous comment I left(about removing comments), everything seems good

**Review by Mila5847 — APPROVED** [review]
Front-end looks good and the code meets the acceptance criteria.

**Review by DenisaNicoletaH — APPROVED** [review]
It looks good and the naming conventions are well done

**Pull request merged**